README.md

```diff
@@ -1,14 +1,44 @@
 # Container Linux Config Transpiler
 
-The config transpiler ("ct" for short) is the utility responsible for transforming a Container Linux instance's configuration from the human-friendly, YAML form into an [Ignition][ignition] configuration. While it is possible to write Ignition configs directly, CoreOS recommends that this tool be used instead. More details about the reasoning and the overall design of Ignition and this project are provided in the [documentation][overview].
+The Config Transpiler ("ct" for short) is the utility responsible for transforming a human-friendly Container Linux Config into a JSON file. This resulting file can be provided to a Container Linux machine when it first boots to provision the machine.
 
-Use the [bug tracker][issues] to report bugs.
+Users of [Matchbox][matchbox] don't need to directly use ct, Matchbox will take ct configs directly.
```

Comment:
![2017-03-20_16 55 52_3200x1800_scrot](https://cloud.githubusercontent.com/assets/2439413/24108535/2af65596-0d8e-11e7-9576-00347dc3689a.png)

I put the mention there on your request. Should I still remove it?